DataConnectors/Qualys VM V2/QualysVM_V2_API_FunctionApp.json

```diff
@@ -0,0 +1,137 @@
+{
+    "id": "QualysVulnerabilityManagementV2",
+    "title": "Qualys Vulnerability Management",
+    "publisher": "Qualys",
+    "descriptionMarkdown": "The [Qualys Vulnerability Management (VM)](https://www.qualys.com/apps/vulnerability-management/) data connector provides the capability to ingest vulnerability host detection data into Azure Sentinel through the Qualys API. The connector provides visibility into host detection data from vulerability scans. This connector provides Azure Sentinel the capability to view dashboards, create custom alerts, and improve investigation ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "QualysHostDetection_CL",
+            "baseQuery": "QualysHostDetection_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Vulerabilities detected",
+            "query": "QualysHostDetection_CL\n | extend Vulnerability = tostring(Results_0_s)\n | summarize count() by Vulnerability\n | top 10 by count_"
```

Comment:
You may replace Results_0_s to QID_s

Reply:
@pbankar, Done. Updated detection query as well: Please review and let me know if any other changes required. Thanks!.